Tests/Microsoft.AppCenter.Test.UWP/UWPCrashesTest.cs

```diff
@@ -9,10 +9,25 @@ namespace Microsoft.AppCenter.Test.UWP
     public class UWPCrashesTest
     {
         [TestMethod]
-        public void VerifyAppLaunchTimestampUTC()
+        public void VerifyManagedErrorLog()
         {
-            ManagedErrorLog managedErrorLog = ErrorLogHelper.CreateErrorLog(new System.Exception("Test Exception"));
-            Assert.AreEqual(managedErrorLog.AppLaunchTimestamp.Value.Kind, DateTimeKind.Utc);
+            try
+            {
+                throw new System.Exception("test exception");
+            }
+            catch (System.Exception e)
+            {
+                // TODO want to check binaries and frames, but this won't be available unless .net native is used.
```

Comment:
Why doesn't the guarded code work unless .NET native is used?  What happens now?

Reply:
We can only populate the exception frames and binaries in .NET native - just a known limitation of the approach. So we can only validate this in release, which uses .NET native.